app/about/contact/page.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react"
+import Image from "next/image"
+import { Hero } from "@/components/Hero"
+import { TextAnimate } from "@/components/magicui/text-animate"
+import { SectionHeader } from "@/components/ui/section-header"
+import { Button } from "@/components/ui/button"
+import { Card, CardContent, CardHeader, CardTitle, CardDescription } from "@/components/ui/card"
+import { cn } from "@/lib/utils"
+import { cardVariants } from "@/components/ui/variants"
+import { FaEnvelopeOpenText, FaTicketAlt, FaBookReader, FaSlack, FaMapMarkerAlt } from "react-icons/fa"
+
+
+const officeHours = [
+  {
+    title: "Computational Biology Core",
+    subtitle: "Join us remotely.",
+    description: "TUESDAYS <br/><br/> 11am - 12pm <br/><br/><br/> WEDNESDAYS <br/><br/> 3pm - 4pm",
+    buttonText: "Find Zoom Link",
+    href: "/ashley-lee",
+  },
+  {
+    title: "Center for Computation and Visualization",
+    subtitle: "Join us remotely or in-person.",
+    description: "FRIDAYS <br/><br/> 10am - 12pm",
+    buttonText: "Find Zoom Link",
+    href: "/ashley-lee",
+  }
+]
+
+const contactUs = [
+  {
+    title: "Email",
+    icon: <FaEnvelopeOpenText/>, 
+    description: "Inquire about resources, support, or a potential collaboration for a research project.",
+    buttonText: "Find Zoom Link",
+    href: "/ashley-lee",
+  },
+  {
+    title: "Submit a Ticket",
+    icon: <FaTicketAlt/>, 
+    description: "Open a ticket with our user services team. We will respond as soon as possible.",
+    buttonText: "Submit a Ticket",
+    href: "/ashley-lee",
+  },
+  {
+    title: "Documentation",
+    icon: <FaBookReader/>, 
+    description: "View Documentation for CCV Services, such as for Stronghold, Oscar, or Globus.",
+    buttonText: "View the Docs",
+    href: "/ashley-lee",
+  },
+  {
+    title: "CCV Slack",
```

Comment:
Ohhhh nooo...... the Slack one i just realized has two buttons on our site. Maybe the terms of service can be a text link in the description? Is that okay?

https://ccv.brown.edu/help

Reply:
i added it as another button!